### DISCUSSION TITLE
Override utilities

### DISCUSSION THREAD

**Discussion by yousefelgoharyx** [tailwindlabs/tailwindcss#8596] [Help]
Is there a way in tailwind plugin system to override some utility classes?
for example if I write a plugin like this
```js
matchUtilities({ mr: (value) => ({ 'margin-left': value }) }, {values} );
```
the `mr` class would still have the margin-right property
I would like to remove it
Is there a way?

Also how can I use match utilities with border radius

**Comment by crswll** [comment]
You can disable core plugins like this:

```js
corePlugins: {
  margin: false,
}
```

This will disable all `m` utilities though (no way to just disable `mr`), so you'll need to account for those.

**Reply by yousefelgoharyx** [reply]
Oh thank you so much 💝
